Sources/Purchasing/StoreKitAbstractions/PromotionalOffer.swift

```diff
@@ -81,41 +81,15 @@ extension PromotionalOffer.SignedData {
                                  timestamp: self.timestamp as NSNumber)
     }
 
-    /// - Throws: ``ErrorCode/unexpectedBackendResponseError`` if the signature cannot be encoded in UTF8
     @available(iOS 15.0, tvOS 15.0, watchOS 8.0, macOS 12.0, *)
     var sk2PurchaseOption: Product.PurchaseOption {
-        get throws {
-            guard let signatureData = self.signature.data(using: .utf8) else {
-                throw ErrorUtils.unexpectedBackendResponse(
-                    withSubError: PromotionalOffer.Error.invalidSignature(self.signature)
-                )
-            }
```

Comment:
cc @aboedo since we talked about this error. No longer needed :)

Reply:
awesome!!

Reply:
there's nothing better than killing error handling because the error can't possibly happen anymore 👏